qupath-extension-openslide/src/main/java/qupath/lib/images/servers/openslide/jna/OpenSlideLoader.java

```diff
@@ -69,6 +69,19 @@ public static synchronized boolean tryToLoad(String... searchPath) throws Unsati
 
     private static synchronized OpenSlideJNA tryToLoadJnaInstance(String... searchPath) {
         String jnaPath = System.getProperty("jna.library.path", null);
+        if (searchPath.length == 0 || (searchPath.length == 1 && searchPath[0].isEmpty())) {
+            logger.info("No search path provided; trying to load OpenSlide from packaged jar");
```

Comment:
This will be logged every time under 'normal' conditions - so I think it should be at the `debug` level.

Also, it *might* not be true - since `jna.library.path` could have been supplied by the user (although probably wasn't).